print-bgp.c

```diff
@@ -80,13 +80,15 @@ struct bgp_open {
     nd_uint16_t bgpo_holdtime;
     nd_uint32_t bgpo_id;
     nd_uint8_t  bgpo_optlen;
+    nd_uint8_t bgpo_opttype;
+    nd_uint16_t bgpo_optlen_extended;
```

Comment:
Two things here:
1. Please line up opttype by adding a space before it, like other nd_uint8_t's
2. Please add a comment something like /* used for extended optional parameters (RFC9072) */

Reply:
Fixed, thanks.